src/runtime/ui-particle-base.ts

```diff
@@ -70,7 +70,7 @@ export class UiParticleBase extends Particle {
 
   renderOutput(...args): void {
     const renderModel = this.render(...args);
-    if (renderModel) {
+    if (Object.keys(renderModel).length > 0) {
```

Comment:
can you exclude this change from this PR? Scott and I are still figuring the right approach here (and I think he had a PR with an alternative fix in flight). 
For now, please, override `shouldRender(...args): boolean {` in RenderNotification.js to return false (i think it would solve the problem).

Reply:
FYI: https://github.com/PolymerLabs/arcs/pull/3752

Reply:
Done.